dapp/README.md

```diff
@@ -32,33 +32,88 @@ Add proposal page of SigmaDAO. On this page user can add a proposal on a existin
 
 ## Setup
 
-Install the dependencies
+This section demonstrates the end-to-end setup required to run the SigmaDAO application fully functioning.
+
+### How to setup postgres?
+
+Download and install the postgres version >= 12 from [here](https://www.postgresql.org/download/).
+
+### How to setup sigmada-indexer?
+
+1. Clone [sigmadao-indexer](https://github.com/scale-it/sigmadao-indexer) repo.
+2. Install go version 1.17 from [here](https://go.dev/dl/).
+3. Build the sigmadao-indexer project:
 
 ```bash
-yarn install
+make
+```
+4. Please make sure postgres server is running.
+5. Sigmadao-indexer can be run on any network using below command:
+
+```bash
+cmd/algorand-indexer/algorand-indexer daemon --data-dir /tmp -P "host=localhost port=5432 user=<postgres-user> password=<postgres-password> dbname=<postgres-database-name> sslmode=disable" --algod-net <link-to-network> --algod-token <network-token> --genesis <path-to-genesis-file>
```

Comment:
What is cmd/algorand-indexer? this is path?

Reply:
It's a command directory where many commands are generated.